src/main/java/zmaster587/advancedRocketry/tile/multiblock/machine/TileChemicalReactor.java

```diff
@@ -151,6 +187,16 @@ public void registerRecipes() {
 		}
 	}
 
+	public static void clearRecipes() {
+		RecipesMachine.getInstance().getRecipes(TileChemicalReactor.class).clear();
+	}
+
+	public static void reloadRecipes() {
+		clearRecipes();
+		LibVulpes.instance.loadXMLRecipe(TileChemicalReactor.class);
+		new TileChemicalReactor().registerRecipes();
+	}
+
```

Comment:
Looks like a dirty fix to me. Isn't it possible to register some kind of special recipe in LibVulpes? If no - it's LibVulpes problem

Reply:
Dirty fix partially cleaned - I now clear and reload JUST the special armor recipes instead of all of the ones for the machine. Still a dirty fix, but better until we figure out a final solution.